src/components/transactions/TxDetails/index.tsx

```diff
@@ -75,7 +78,7 @@ const TxDetailsBlock = ({ txSummary, txDetails }: TxDetailsProps): ReactElement
         )}
 
         <div className={css.txSummary}>
-          {isUnsigned && <UnsignedWarning />}
+          {isTrusted === false && <UnsignedWarning />}
```

Comment:
This is also not correct because if `isMultisigDetailedExecutionInfo(txDetails.detailedExecutionInfo)` returns false, you will show the warning.